common/achievement/Readme.md

```diff
@@ -1,4 +1,192 @@
 # Achievement
 
-Achievements are a part of the Gamification.
-Documentation follows...
+Achievements are a part of the Gamification. They are stored in the database as user_achievement and should not be mistaken with the achievement_template database table. The distinct differences are explained in Chapter [1.1 Database Models](#database-models)
+
+### Table of Contents
```

Comment:
In my opinion we should also describe the concept of the whole achievement system, because we can't link the FigJam File of the concept and we should use this documentation as single source. Maybe we can add another markdown file for `Metric.md` and `Bucket.md` (maybe for `Condition.md`, because we have to explain how the condition should be defined), that we can explain what the idea behind the concepts is. So that we put all informations from the concept into the documentation.

I think this diagram shows the best (thanks to Daniel) how buckets work.


```
 . . . . . . . .    -> Raw events
|__||__||__||__|    -> Buckets based on type
 \ / \ / \ /\ /     -> Aggregate bucket events to single value
  .   .   .  .      -> Aggregate values
   \        /
    \      /
       .     -> result for condition
```

Reply:
- [ ] A Graphic for Bucket aggregation was introduced to the documentation
- [ ] An explanation for metrics was added to the documentation
- [ ] The concept for conditions and how to write them was added

Reply:
I would still keep this all in one file. The advantage of which offering the ability to reference the chapters, where to find information e.g about the buckets, with a hyper reference. This could be very beneficial when searching for information in the documentation.